src/Droppable/README.md

```diff
@@ -29,7 +29,7 @@ Check out [Draggables API](../Draggable#api) for the base API
 
 Check out [Draggables options](../Draggable#options) for the base options
 
-**`dropzone {String|HTMLElement[]|NodeList|Function}`**  
+**`dropzone {String|HTMLElement[]|NodeList|Function}`**
```

Comment:
We'll need the two trailing spaces here 😬

Reply:
What is the purpose of the trailing spaces?

Reply:
It adds a new line. It's not super crucial though, we can also easily add it back

Reply:
There has to be some other way to achieve that... does a hard return not work for some reason?

Reply:
I remember trying that, and it didn't work. Someone at work mentioned that you can achieve the effect with two trailing spaces. I also added a rule for `editorconfig`: https://github.com/Shopify/draggable/blob/master/.editorconfig#L15

Reply:
But agreed that it's not great, easy to overlook the syntax

Reply:
Double trailing spaces are part of the markdown spec, otherwise the output is wrapped.  The only alternative for parsers that recognize this is to add a double newline, but this is ugly in a list.